edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -40,29 +41,45 @@ class EnsureJWTAuthSettingsMiddleware(MiddlewareMixin):
     """
     _required_permission_classes = (NotJwtRestrictedApplication,)
 
+    @staticmethod
+    def _get_decomposed_permissions(permission):
+        """
+        Given a composed permission class or instance, return its component parts.
+
+        Examples:
+
+        Composition using DRF native support in 3.9+:
```

Comment:
This comment could be a `#` comment next to moved code. See later comment about moving `decomposed_permissions` code.